.github/licenserc.yaml

```
@@ -14,3 +14,5 @@ header:
     - "**/*.yaml"
     - "**/*.toml"
     - "**/*.lock"
+    - "**/*.cert"
```

```suggestion
    - "**/*.crt"
```

I notice that the new file name is `server.crt`